easy/test_is_up.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python2
+
+import time
+from urllib import urlopen
+
+
+def wait_for(predicate, timeout=300):
+    amount_slept = 0
+    while not predicate():
+        time.sleep(0.5)
+        amount_slept += 0.5
+        if amount_slept >= timeout:
+            raise SystemExit('Timeout waiting for ' + predicate.__name__)
```

Comment:
I remember I used recently a Timeout exception, would that be more appropriate in this context?

Reply:
I think it doesn't exist in python2